assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -57,10 +57,26 @@ class AgendaListItemIcons extends React.Component {
         };
     }
 
+    getItemByschema(props) {
+        const {listConfig} = props;
+        if (listConfig && listConfig.subject && listConfig.subject.scheme) {
```

Comment:
We should probably support the ability to show ALL subjects.
Meaning, if `listConfig.subject.scheme` is defined, then we filter the subjects, otherwise we return all subjects if `listConfig.subject` is defined

Reply:
Okay